content/docs/guides/contribute/preamble.fr.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Avant toutes choses"
+linkTitle: "Préambule"
+weight: 1
+description: Bonjour ! Quelques premières informations importantes.
+---
+
+Déjà, merci d'avoir pris le temps de contribuer !
+
+Si les sections suivantes forment un guide du contributeur, ce n'est pas pour autant un ensemble de règles strictes. En fait, si vous avez déjà contribué à des gros projets opensource, la suite ne sera pas surprenante. Dans le cas contraire, elle vous sera probablement très utile !
+
+### Communiquer
+
+Vous pouvez gagner du temps en discutant de votre projet de contribution avec les autres contributeurs :
+
+- Rejoignez le salon de messagerie instantanée sur IRC à [libera.chat#osrd](https://web.libera.chat/#osrd).
```

Comment:
```suggestion
```

We're not using this channel anymore (I'm still on it, but I only check it once every few months), I don't this we should keep displaying it unless we're really using it

Reply:
Removed